doc/decisions/frontend_keyhandling.md

```diff
@@ -0,0 +1,65 @@
+# Frontend Keybindings
+
+## Problem
+
+In the development of PermaplanT, we have identified the need for a robust keybinding system to enhance the user experience. Keybindings are utilized in various parts of the application, and we need to establish a cohesive approach for their implementation.
+
+## Constraints
+
+Our keybinding solution should adhere to the following constraints:
+
+- **Ease of Integration**: The keybinding system should be straightforward to integrate into different components of the application.
+
+- **User Configuration**: We should plan for future functionality that allows users to configure keybindings according to their preferences.
```

Comment:
This is not needed. It should only be developer-friendly: developers should easily get to know where which key binding is assigned.

Having all features to great good UX/UI is however needed (e.g. scoping, maybe @tschawax has more input on that).